crates/sound/src/vhu_sound.rs

```diff
@@ -188,7 +188,7 @@ impl VhostUserSoundThread {
         //iterate over each sound request
         for desc_chain in requests {
             let descriptors: Vec<_> = desc_chain.clone().collect();
-            debug!("Sound request with n descriptors: {}", descriptors.len());
+            //debug!("Sound request with n descriptors: {}", descriptors.len());
```

Comment:
There are various changes commenting debug statements, not sure if those are meant to be committed

Reply:
no no, they're not meant to be committed, this was just for testing. I would clean things up. Thanks :)